<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureMange</class>
 <widget class="QWidget" name="TextureMange">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1880</width>
    <height>1019</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1880</width>
    <height>1019</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1880</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PSDMerge_v1.0</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="statusTip">
       <string notr="true"/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>PsdTexture</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>720</y>
         <width>901</width>
         <height>251</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>输出设置</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,0,0">
          <item>
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item>
           <widget class="QPushButton" name="coust_Button">
            <property name="text">
             <string>........</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>自定义路径（开发中）</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="tex_json">
            <property name="text">
             <string>保存名称预设</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>重置</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>文件名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="basename">
            <property name="text">
             <string>T</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="r_name">
            <property name="text">
             <string>_Rock</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="g_name">
            <property name="text">
             <string>_01</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="b_name">
            <property name="text">
             <string>_m</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="a_name">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tex_file_line">
            <property name="text">
             <string>.tga</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>删除拆解的PNG</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="mergeButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial Narrow</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>合并输出</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>901</width>
         <height>151</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Psd</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>PSD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="psd_lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="psd_Button">
            <property name="text">
             <string>........</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0,1">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>注：默认PNG拆分到psd目录中</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>按照标签拆分需要通道</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>标签自动导入</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="out_Button">
            <property name="font">
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text">
             <string>PSD分解输出</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar1">
          <property name="maximum">
           <number>105</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="show_box">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>10</y>
         <width>931</width>
         <height>961</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>BIZ UDPGothic</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>预览</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,8,1">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_image">
          <property name="text">
           <string>图片</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pngname_lineEdit">
          <property name="text">
           <string>名字</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>901</width>
         <height>541</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>RGBA</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="yushe01">
            <item>
             <widget class="QPushButton" name="mark_Button">
              <property name="text">
               <string>保存标签名称预设</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mark_Button_3">
              <property name="text">
               <string>保存颜色设置预设</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>R</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1,0">
             <item>
              <widget class="QLabel" name="R">
               <property name="cursor">
                <cursorShape>ArrowCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 0, 0);</string>
               </property>
               <property name="text">
                <string>R：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mark_r">
               <property name="text">
                <string>金属 </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="r_lineEdit">
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 0, 0);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="r_pushButton">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>........</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="autoRepeat">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2,1,2,8">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>默认加载颜色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="default_r">
               <property name="text">
                <string>#000000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>合并颜色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="merge_r_c">
               <property name="text">
                <string>#000000</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>G</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="g_frame" stretch="0,0,1,0">
             <item>
              <widget class="QLabel" name="G">
               <property name="cursor">
                <cursorShape>ArrowCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(39, 204, 20);</string>
               </property>
               <property name="text">
                <string>G：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mark_g">
               <property name="text">
                <string>内描边</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="g_lineEdit">
               <property name="styleSheet">
                <string notr="true">color: rgb(39, 204, 20);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="g_pushButton">
               <property name="text">
                <string>........</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="g2_frame" stretch="1,2,1,2,8">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>默认加载颜色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="default_g">
               <property name="text">
                <string>#000000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>合并颜色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="merge_g_c">
               <property name="text">
                <string>#808080</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>B</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="b_frame" stretch="0,0,1,0">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="cursor">
                <cursorShape>ArrowCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 166, 255);</string>
               </property>
               <property name="text">
                <string>B：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mark_b">
               <property name="text">
                <string>高光</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="b_lineEdit">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 166, 255);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="b_pushButton">
               <property name="text">
                <string>........</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="b2_frame" stretch="1,2,1,2,8">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>默认加载颜色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="default_b">
               <property name="text">
                <string>#000000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>合并颜色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="merge_b_c">
               <property name="text">
                <string>#808080</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>A</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="a_frame" stretch="0,0,1,0">
             <item>
              <widget class="QLabel" name="A">
               <property name="cursor">
                <cursorShape>ArrowCursor</cursorShape>
               </property>
               <property name="text">
                <string>A：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mark_a">
               <property name="text">
                <string>外描边</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="a_lineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="a_pushButton">
               <property name="text">
                <string>........</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="a2_frmae" stretch="1,2,1,2,8">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>默认加载颜色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="default_a">
               <property name="text">
                <string>#000000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>合并颜色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="merge_a_c">
               <property name="text">
                <string>#808080</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>More</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
